Makefile

```diff
@@ -38,14 +39,14 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # appstudio.redhat.org/service-provider-integration-operator-bundle:$VERSION and appstudio.redhat.org/service-provider-integration-operator-catalog:$VERSION.
-SPIO_IMAGE_TAG_BASE ?= appstudio.redhat.org/service-provider-integration-operator
+SPIO_IMAGE_TAG_BASE ?= quay.io/skabashn/service-provider-integration-operator
```

Comment:
is this correct image?

Reply:
for now yes, eventually we will go to app-studio quay account.